pkg/install/resource/tm_namespace.go

```diff
@@ -0,0 +1,44 @@
+package resource
+
+import (
+	"context"
+
+	"github.com/datawire/ambassador/pkg/kates"
+)
+
+type nsResource int
+
+var TrafficManagerNamespace Instance = nsResource(0)
+var TrafficManagerNamespaceKeep Instance = nsResource(1)
```

Comment:
Can we change these from `var` to `const`?

Reply:
No, because they are of type `Instance` which is an interface. We could of course cover that with an anonymous var instead.

Reply:
I'll replace with:
```
const TrafficManagerNamespace = nsResource(0)
const TrafficManagerNamespaceKeep = nsResource(1)

var _ Instance = TrafficManagerNamespace
```
and do the same thing for the other integer base types.